### PR TITLE
Allow saving while source edit mode is open.

### LINKED ISSUE #2274: Hitting update while in source editor doesn't save changes done in HTML.
# Problem 

Changes to markup are lost if you hit "update" to a text area block in source view.

Screen Shot 2020-10-01 at 9 38 49 AM

# Proposed Solution

?

### PR DESCRIPTION
**Pull request by cory-skeers**
Resolves #2274 

# Problem
When adding or updating a block with filtered or minimal html text field, if the source edit is open, changes will not be saved.

## Steps to reproduce

From issue: https://github.com/uiowa/uiowa/issues/2274
Add block to section
Create custom block
Edit text in body field
Save block
Edit block
Switch body field to ckeditor source mode
Edit text
Save
Changes are NOT saved!
Once I changed back to non-source mode before saving the changes are saved properly.

## To test
Try the above with the edited JS fix from this branch. Check that updates are now saved.

### CODE REVIEW

File: docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js [hunk 1]
```diff
@@ -32,4 +32,28 @@
           });
         }
     };
+
```
Comment (pyrello) [inline docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js:35]: Is this specific to layout builder? If not, I think we should maybe not add it through this module. Possibly via uiowa_core?

**Review by pyrello — CHANGES_REQUESTED** [review]

File: docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js (hunk 1 above)
Comment (cory-skeers) [reply docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js:35]: I believe it is specific to layout builder, but I will check!

Comment (cory-skeers) [reply docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js:35]: @pyrello Tested editing a node body field and the footer contact info block, and both maintain changes when saving with the source edit form open, where text fields edited through layout builder do not. Looks to be limited to ckeditor within layout builder.

Comment (pyrello) [reply docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js:35]: Sounds like this is appropriately scoped. Thanks!

**Review by GaryRidgway — APPROVED** [review]
I have tested  this functionality and can confirm that when using a CKEditor WYSIWYG i can now save in source view

**Review by pyrello — APPROVED** [review]
Changing this, since Alan reviewed.

**Pull request merged**